Frameworks/CoreGraphics/DWriteFontCollectionHelper.mm

```diff
@@ -130,6 +130,32 @@
     return S_FALSE;
 }
 
+HRESULT DWriteFontCollectionHelper::GetAvailableFonts(std::vector<ComPtr<IDWriteFont>>& outFontList) {
```

Comment:
It's sort of a "out param pointer" with a little bit less room for error, and though it's not ideal I'm not sure there's a better way to pass around a dynamic list of IDWriteFonts